docs/performance.md

```diff
@@ -24,3 +24,19 @@ average rate = 9717.58 msg/sec, count=1749420, time=180.026, (average) msg size=
 
 Splunk Enterprise's implementation of the http event collection server responds to the client with a status code 200 and fails to commit the events to disk during a rolling restart. In our testing, 20-30 events per indexer are lost.
 
+## Guidance on sizing hardware
+
+The following reference deployment hardware specifications are based on Splunk performance testing results in AWS 
+The overall load on your deployment HW will vary based on the percentage of events which are not handled by a filter or 
+custom filters added not accounted for here. While we consider the following conservative actual hardware performance will vary
```

Comment:
How about this wording instead?...
"The overall load on your deployment HW will vary based on the percentage of events not handled by a filter or use of exceptionally complex regex in filters."